### ISSUE TITLE
OSM Could Not Determine A Route 

### ISSUE THREAD

**Issue by seniorm0ment** [osmandapp/OsmAnd#16616]
I'm getting an error when trying to route. It shows the progress bar and just gets stuck. It's saying 
OSM Could not determine a route for your settings. Maybe some apps are not loaded, you can run an online check for the necessary maps."

If I click check for necessary maps,  it seems to fail.
I have online maps, and I have offline maps downloaded to the device. I have reception, and I have tried rebooting the device and even clearing app cache. Does not fix it.

It seems to work sometimes and fail othertimes.

 OSMAnd Fdroid 4.3.8

**Comment by TheSnoozer** [comment]
Thanks for reporting, could you please clarify:

1) Route, start and finish coordinates
2) The presence of traffic restrictions
3) Your profile (e.g. car?)
4) Last time maps had been updated

**Comment by seniorm0ment** [comment]
1. Start: 41.76101 -71.45835; End: 41.82829 -71.38815
2. No restrictions, Consider temporary limitations enabled.
3. Profile: what specifically? Vehicle paramaters (weight and stuff) set to none. But yes its my car.
4. Last time the offline maps updated for this state was jan 31st. Also tried running live update when i was having that issue, then rebooting and it didnt fix

**Comment by wolfsolver** [comment]
On 4.4.1, same parameters, maps fresh download route is calculated.

![Screenshot_20230311-183910.png](https://user-images.githubusercontent.com/3205448/224503420-ad875b2f-1a78-4717-972e-7bfe5eca7bfb.png)

Hope this is useful

**Comment by seniorm0ment** [comment]
Interesting. Also I did encounter this again with another route recently, I forget the start and destination but it is very aggravating because I only use gps when I don't know where I'm going generally so it is frustrating not to have it when I really do need it.

And like I said originally, this has happened more than once already.
Sometimes it works no issues, sometimes it doesn't. Had a hard time determining what the issue may be just by playing around, it almost seems random.

**Comment by Zirochkabila** [comment]
@seniorm0ment The route calculation took about 5s - please download OsmAmd from the official app store and try again

<table style="width:100%">
  <tr>
<th><img src="https://user-images.githubusercontent.com/104760013/224559641-c82517f4-e9c0-4c18-850d-40d62da526d5.jpeg"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by vshcherb** [comment]
Not reproduced...

**Comment by seniorm0ment** [comment]
Encountering this again.
I noticed routing was using offline routing, so I set it to use one of the online ones, rebooted the app and it routed no issues. Haven't used much since. But I fail to see why there would be issues with offline routing. I could understand online routing having issues.

Also if I use online routing, what happens when I lose internet/enter an area with poor internet? Does it get lost? Or just fallback to offline?

**Comment by TheSnoozer** [comment]
On what version of OSMAnd do you encounter this issue? Have you tried the latest as suggested above?

**Comment by seniorm0ment** [comment]
Yes I was encountering issues when testing with the official version. I ended up just reinstalling the fdroid version because I do not wish to use Play Services. That being said, it hasn't done it for a bit and then it started doing it again. 
Is there a way to view logs?
